Lecture-security.md

```diff
@@ -181,6 +182,15 @@ to automatically use only HTTPS, which prevents hackers from performing this
 sort of man-in-the-middle attack.
 ```
 
+### Symantec ISTR (2019)
+
+The [Internet Security Threat Report](https://www.symantec.com/content/dam/symantec/docs/reports/istr-24-2019-en.pdf) (ISTR) published by Symantec outlines the threat landscape as seen in early 2019. They determine the trends using telemetry collected from over 123 Million sensors that monitor activities of over 300,000 organizations that use Symantec’s protection. We look at a few relevant trends:
+
+1. In general, web attacks on endpoints (host machines) have increased by 56%. Spear phishing is the most prevalent infection vector used by the attackers, where they are more likely to use malicious attachments rather than malicious URLs. Nearly half of the malicious email attachments are Microsoft Office files, which are disguised either as invoices or email delivery-failure notifications.
```

Comment:
It says that web attacks have increased by 56%. It is unclear what the two datapoints here are: was this computed between 2018 and 2019? Or some other time period?

Reply:
Web attacks have increased by 56% in the year 2018. This is compared to the attacks that happened in 2017. They state that they analyzed "over 1.5 Billion web requests each day". 348 Million web attacks were blocked in total, averaging 953 thousand attacks per day.

This also holds for the "1 in 10 URLs" statistic. In 2017, 1 in 16 URLs were malicious. That's an increase of 3.4% from the previous year. (Note: This could be because of a change in the data distribution but they don't mention that)